2share/7.0/community-7.0.1/overload.gradle

```diff
@@ -3,13 +3,13 @@ ext {
             version: [
                     major: 7,
                     minor: 0,
-                    rev: '0'
+                    rev: '1'
```

Comment:
Needs to have a label added.

See https://github.com/xenit-eu/docker-share/commit/768e4771142cadafd35859e88a902f91e37b4132 for example